src/python/dxpy/bindings/dxfile_functions.py

```diff
@@ -288,7 +288,7 @@ def verify_part(_part_id, got_bytes, hasher):
 
         return True
 
-def upload_local_file(filename=None, file=None, media_type=None, keep_open=False,
+def upload_local_file(filename=None, file=None, media_type=None, keep_open=False, write_buffer_size=None,
```

Comment:
New keyword args should be specified after any existing keyword args for compatibility with existing callers.